Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -297,11 +297,11 @@ - (void)addNotificationListeners
     [[NSNotificationCenter defaultCenter] addObserver:self
                                              selector:@selector(yapDatabaseModified:)
                                                  name:YapDatabaseModifiedNotification
-                                               object:nil];
+                                               object:TSStorageManager.sharedManager.dbNotificationObject];
```

Comment:
Filter by database so that the database copying process doesn't spam the views with irrelevant modifications.